react/week2/week2-react-modified/src/components/TodoItems/TodoItems.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import "./TodoItems.css";
+
+const TodoItems = (props) => {
+  const { items } = props;
+  const { deletedItem, completedItem } = props;
+  let length = items.length;
+  const ListItems = length ? (
+    items.map((item) => {
+      return (
+        <div className="ListItems" key={item.id}>
+          <span
+            className={
+              item.completed
+                ? "description list-item completed"
+                : "description list-item"
+            }
+          >
+            {item.description}
+          </span>
+          <span
+            className={
+              item.completed ? "when list-item completed" : "when list-item"
+            }
+          >
+            {item.when}
```

Comment:
```suggestion
            new Date(item.when).toDateString()
```
to have same format for all dates